webpack/assets/javascripts/react_app/components/common/table/components/TableBody.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Table as PfTable } from 'patternfly-react';
+
+import TableBodyMessage from './TableBodyMessage';
+
+const TableBody = ({ columns, rows, message, ...props }) => {
+  if (message) {
+    return (
+      <TableBodyMessage colSpan={columns.length}>{message}</TableBodyMessage>
+    );
+  }
+
+  return (
+    <PfTable.Body rows={rows} rowKey={({ rowIndex }) => rowIndex} {...props} />
+  );
+};
+
+TableBody.propTypes = {
```

Comment:
I'm not sure where this is coming from (pf?) but I get this error in my console:
```
checkPropTypes.js:19 Warning: Failed prop type: The prop `children` is marked as required in `EllipisWithTooltip`, but its value is `null`.
    in EllipisWithTooltip (created by BodyRow)
    in BodyRow (created by Body)
    in tbody (created by Body)
    in Body (created by TableBody)
    in TableBody (created by Table)
    in table (created by Provider)
    in Provider (created by TablePfProvider)
    in TablePfProvider (created by Table)
    in div (created by Table)
    in Table (created by ModelsTable)
    in LoadingState (created by ModelsTable)
    in ModelsTable (created by Connect(ModelsTable))
    in Connect(ModelsTable) (created by I18nProviderWrapper(Connect(ModelsTable)))
    in IntlProvider (created by I18nProviderWrapper(Connect(ModelsTable)))
    in I18nProviderWrapper(Connect(ModelsTable)) (created by StoreProvider(I18nProviderWrapper(Connect(ModelsTable))))
    in Provider (created by StoreProvider(I18nProviderWrapper(Connect(ModelsTable))))
    in StoreProvider(I18nProviderWrapper(Connect(ModelsTable))) (created by DataProvider(StoreProvider(I18nProviderWrapper(Connect(ModelsTable)))))
    in DataProvider(StoreProvider(I18nProviderWrapper(Connect(ModelsTable))))
```

Reply:
I couldn't reproduce it. I guess it's because I am not checking the `value` in `common/table/formatters/ellipsisCellFormatter.js`. Let me know if it is fixed.